tools/utils/index.ts

```diff
@@ -118,9 +118,10 @@ export function lowerCaseAddresses(obj: any): any {
   for (const key in obj) {
     if (typeof obj[key] === 'object') {
       lowerCaseAddresses(obj[key])
-    }
-    if (typeof obj[key] === 'string' && obj[key].indexOf('0x') === 0) {
+    } else if (typeof obj[key] === 'string' && obj[key].indexOf('0x') === 0) {
```

Comment:
what's the purpose for the else addition?

Reply:
Only checks the second condition if the first fails, better logic.